doc/tasks/deptrac.md

```diff
@@ -14,65 +14,23 @@ grumphp:
         deptrac:
             depfile: ~
             formatter: ~
-            graphviz_display: ~
-            graphviz_dump_image: ~
-            graphviz_dump_dot: ~
-            graphviz_dump_html: ~
-            junit_dump_xml: ~
-            xml_dump: ~
-            baseline_dump: ~
+            output: ~
 ```
 
 **depfile**
 
 *Default: null*
 
-Set path to deptrac configuration file. Example: `/var/www/src/depfile.yml`
+Set path to deptrac configuration file. Example: `/var/www/src/deptrac.yaml`
```

Comment:
Isn't this still depfile? https://qossmic.github.io/deptrac/#the-depfile

Reply:
No. https://github.com/qossmic/deptrac/blob/main/UPGRADE.md